### ISSUE TITLE
components are created but with no content

### ISSUE THREAD

**Issue by TianCNCA** [vaadin/vaadin-grid#399]
I'm trying to use vaadin with [angular2-seed](https://github.com/mgechev/angular2-seed).

I've followed the instructions on the [Angular2 Integration] (https://vaadin.com/docs/-/part/elements/angular2-polymer/tutorial-index.html) page, however the components are not built properly. By which I mean on the page I can see DOM elements like `<vaadin-grid>` and the objects bounded to the `items` property, but there is no data in the `table` element inside the `<vaadin-grid>` and nothing is showing on the page, not even the frame.

The full HTML code is like below:

```
<spa-grid _nghost-yjd-3=""><vaadin-grid frozen-columns="0" class="x-scope vaadin-grid-0" _ngcontent-yjd-3="" ng-reflect-items="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]">
    <iframe id="measureobject" class="vaadin-grid style-scope" src="about:blank"></iframe>
  <div class="vaadin-grid style-scope vaadin-grid style-scope vaadin-grid" tabindex="0" style="height: 100%; width: 100%;">
    <div tabindex="-1" class="vaadin-grid style-scope vaadin-grid-scroller vaadin-grid style-scope vaadin-grid-scroller-vertical" style="padding-right: 17px; width: 0px; height: 210px; overflow-y: scroll; top: 56px;">
      <div class="style-scope vaadin-grid" style="width: 17px; height: 480px;">
      </div>
    </div>
    <div tabindex="-1" class="vaadin-grid style-scope vaadin-grid-scroller vaadin-grid style-scope vaadin-grid-scroller-horizontal" style="display: none; padding-bottom: 17px; height: 0px; width: 1886px; left: 0px;">
      <div class="style-scope vaadin-grid" style="height: 17px; width: 0px;">
      </div>
    </div>
    <div class="vaadin-grid style-scope vaadin-grid-tablewrapper" style="height: 266px; width: 1886px;">
      <table class="style-scope vaadin-grid">
        <thead class="vaadin-grid style-scope vaadin-grid-header">
          <tr class="vaadin-grid style-scope vaadin-grid-row" style="width: 0px;">
          </tr>
        </thead>
        <tbody class="vaadin-grid style-scope vaadin-grid-body" style="margin-top: 56px; transform: translate3d(0px, 0px, 0px);">
          <tr class="vaadin-grid style-scope vaadin-grid-row vaadin-grid-row-has-data" style="width: 0px; transform: translate3d(0px, 0px, 0px);">
          </tr>
          <tr class="vaadin-grid style-scope vaadin-grid-row vaadin-grid-row-has-data vaadin-grid-row-stripe" style="width: 0px; transform: translate3d(0px, 48px, 0px);"></tr><tr class="vaadin-grid style-scope vaadin-grid-row vaadin-grid-row-has-data" style="width: 0px; transform: translate3d(0px, 96px, 0px);">
          </tr>
          <tr class="vaadin-grid style-scope vaadin-grid-row vaadin-grid-row-has-data vaadin-grid-row-stripe" style="width: 0px; transform: translate3d(0px, 144px, 0px);"></tr>
          <tr class="vaadin-grid style-scope vaadin-grid-row vaadin-grid-row-has-data" style="width: 0px; transform: translate3d(0px, 192px, 0px);"></tr><tr class="vaadin-grid style-scope vaadin-grid-row vaadin-grid-row-has-data vaadin-grid-row-stripe" style="width: 0px; transform: translate3d(0px, 240px, 0px);"></tr>
        </tbody>
        <tfoot class="vaadin-grid style-scope vaadin-grid-footer"></tfoot>
      </table>
    </div>
    <div class="vaadin-grid style-scope vaadin-grid-header-deco" style="width: 17px; height: 56px;"></div><div class="vaadin-grid style-scope vaadin-grid-footer-deco" style="width: 17px;"></div><div class="vaadin-grid style-scope vaadin-grid-horizontal-scrollbar-deco" style="display: none; height: 17px;"></div></div>
    <table _ngcontent-yjd-3="">
        <colgroup _ngcontent-yjd-3=""> 
            <col _ngcontent-yjd-3="" name="id">
            <col _ngcontent-yjd-3="" name="name">
        </colgroup>
    </table>
</vaadin-grid>
</spa-grid>
```

As you can see, the vaadin components are created but without data. I would really like to know why it is behaving like this and any help is appreciated.

Below are my TypeScript files:

grid.component.html:

```
<vaadin-grid [items]="shifts">
    <table>
        <colgroup>
            <col name="id">
            <col name="name">
        </colgroup>
    </table>
</vaadin-grid>
```

grid.component.ts:

```
import { Component, OnInit } from '@angular/core';
import { PolymerElement } from '@vaadin/angular2-polymer';
import { Shift, ShiftService } from '../shared/shift/index';

/**
 * This class represents the grid component.
 */
@Component({
    moduleId: module.id,
    selector: 'spa-grid',
    templateUrl: 'grid.component.html',
    styleUrls: ['grid.component.css'],
    directives: [PolymerElement('vaadin-grid')]
})
export class GridComponent implements OnInit {
    shifts: Shift[] = [
        { id: 11, name: 'Mr. Nice' },
        { id: 12, name: 'Narco' },
        { id: 13, name: 'Bombasto' },
        { id: 14, name: 'Celeritas' },
        { id: 15, name: 'Magneta' },
        { id: 16, name: 'RubberMan' },
        { id: 17, name: 'Dynama' },
        { id: 18, name: 'Dr IQ' },
        { id: 19, name: 'Magma' },
        { id: 20, name: 'Tornado' }
    ];

    constructor(private _shiftService: ShiftService) { }

    getShifts() {
        this._shiftService.getShifts().then(shifts => this.shifts = shifts);
    }

    ngOnInit() {
        //this.getShifts(); // I thought it was because this line of code so I hard-coded shifts but it didn't work
        console.log(this.shifts);
    }
}
```

**Comment by tomivirkki** [comment]
Hi @TianCNCA,

This is actually an issue with angular2-polymer, not vaadin-grid. Angular 2 seed imports Angular 2 umd bundles instead of the separate source files and angular2-polymer 1.0.0-beta1 only targets the src scope when setting a custom DOM adapter. This will be fixed by https://github.com/vaadin/angular2-polymer/pull/44 in the next beta release.

Anyway, you can get and Angular 2 seed project running with angular2-polymer 1.0.0-beta1 as well if you change
`node_modules/@angular/platform-browser/bundles/platform-browser.umd.js`
to
`node_modules/@angular/platform-browser/index.js`
in the seed.config.ts file

**Comment by tomivirkki** [comment]
Fixed in https://github.com/vaadin/angular2-polymer/pull/44

You can now use the newly released angular2-polymer 1.0.0-beta2 in your project without needing any changes
